charms/jupyter-ui/tests/integration/test_charm.py

```diff
@@ -68,3 +69,41 @@ async def test_ui_is_accessible(ops_test: OpsTest):
     assert result_status == 200
     assert len(result_text) > 0
     assert "Jupyter Management UI" in result_text
+
+
+@pytest_asyncio.fixture
+async def deploy_kubeflow_dashboard(ops_test: OpsTest):
+    """Deploys kubeflow-dashboard and kubeflow-profiles, cleaning them up after the usage."""
+    kubeflow_profiles = "kubeflow-profiles"
+    await ops_test.model.deploy(kubeflow_profiles, channel="stable", trust=True)
+
+    kubeflow_dashboard = "kubeflow-dashboard"
+    # Requires latest/edge until https://github.com/canonical/kubeflow-dashboard-operator/pull/134
+    # is merged into latest/stable
+    ops_test.model.deploy(kubeflow_dashboard, channel="edge", trust=True)
+
+    await ops_test.model.relate(kubeflow_dashboard, kubeflow_profiles)
+
+    await ops_test.model.wait_for_idle(
+        apps=[kubeflow_dashboard, kubeflow_profiles], status="active", timeout=60 * 5
+    )
+
+    yield kubeflow_dashboard
+
+    # We should remove things, but removal was flaky during initial testing and `block_until_done`
+    # does not block successfully.  Try this again once dashboard and profile use chisme
+    # KRH.delete to see it they work better
+    # ops_test.model.remove_application(kubeflow_dashboard, block_until_done=True)
+    # ops_test.model.remove_application(kubeflow_profiles, block_until_done=True)
+
+
+async def test_dashboard_link_relation(ops_test: OpsTest, deploy_kubeflow_dashboard: str):
```

Comment:
I dont know about this test. Cant we curl and just fulltext search for the link? 

If not can't we check if the relation data is in databag (not sure if we do that somewhere)

Reply:
Removed based on discussions in the team.  This will be covered by https://github.com/canonical/bundle-kubeflow/issues/636